deploy/cr.yaml

```
@@ -5,8 +5,12 @@ metadata:
 spec:
   allowUnsafeConfigurations: false
 #  pause: false
+  crVersion: 0.2.0
   secretsName: cluster1-secrets
   sslSecretName: cluster1-ssl
+  #upgradeOptions:
+  #  versionServiceEndpoint: http://percona-version-service:80
```

https://check.percona.com
```suggestion
  #  versionServiceEndpoint: https://check.percona.com
```

and we need uncomment all these options

---

https://github.com/percona/percona-server-mysql-operator/pull/151/commits/2336bee0e7c97ad95cfa63e3e3f05995cd11864f